### PR TITLE
[SFSRAP-39] Added /acceptLicense arg processing

### PR DESCRIPTION
**Pull request by duncanpMS**
TODO: use the supplied argument

### CODE REVIEW

File: RoslynPluginGenerator/CommandLine/ProcessedArgs.cs [hunk 1]
```diff
@@ -13,8 +13,9 @@ public class ProcessedArgs
         private readonly NuGetReference analyzerRef;
         private readonly string sqaleFilePath;
         private readonly string language;
+        private readonly bool acceptLicenses;
 
-        public ProcessedArgs(NuGetReference analyzerRef, string sqaleFilePath, string language)
+        public ProcessedArgs(NuGetReference analyzerRef, string sqaleFilePath, string language, bool acceptLicenses)
```
Comment (HSAR) [inline RoslynPluginGenerator/CommandLine/ProcessedArgs.cs]: Is there a better way to handle arguments that doesn't involve an ever-increasing number of parameters? I can see this getting out of hand as we add options for richer authoring experiences.

Comment (duncanpMS) [inline RoslynPluginGenerator/CommandLine/ProcessedArgs.cs]: That's a very general question more suited to a discussion than a PR. Are there any specific issues you want raised about this PR, or are you saying that three parameters is already too many?

Comment (HSAR) [inline RoslynPluginGenerator/CommandLine/ProcessedArgs.cs]: We can discuss this offline.

**Comment by HSAR** [comment]
LGTM

**Pull request merged**